src/extension/background-script/actions/ln/makeInvoice.js

```diff
@@ -2,13 +2,11 @@ import PubSub from "pubsub-js";
 import state from "../../state";
 
 const makeInvoice = async (message, sender) => {
-  if (message.args.memo === undefined) {
-    message.args.memo = "Alby invoice memo";
-  }
+  const memo = memo.args.memo || memo.args.defaultMemo || "Alby invoice memo";
+  message.args.memo = memo;
```

Comment:
```suggestion
```
I think we do not even need that. we only use `amount` and `memo` in line 12 and ff. 
or am I missing the message usage somewhere?